source/help/messaging/attaching-files.rst

```diff
@@ -25,22 +25,20 @@ Click the thumbnail of an attached file to open it in the file previewer.
 
 Downloading Files
 ~~~~~~~~~~~~~~~~~
-You can download an attached file by:
-
--  Clicking the download icon next to the file thumbnail
+You can download an attached file by clicking the download icon next to the file 
+thumbnail.
 
 Sharing Public Links
 ~~~~~~~~~~~~~~~~~~~~
-Public links allow you to share attachments with people outside your 
-Mattermost team.
-
--  Click the thumbnail of an attachment > click **Get Public Link**
+Public URL's allow you to share attachments with anyone outside the Mattermost 
```

Comment:
Thank you for all the review you have done on this document, the comments you have made have been really helpful!